openapi-bal-task-plugin/src/main/java/io/ballerina/openapi/bal/tool/OpenAPICodeGeneratorTool.java

```diff
@@ -190,12 +209,18 @@ private OpenAPI getOpenAPIContract(Path ballerinaFilePath, Path openAPIPath, Loc
                 File openapiContract = new File(file, openAPIPath.toString());
                 relativePath = Paths.get(openapiContract.getCanonicalPath());
             }
-            return normalizeOpenAPI(Path.of(relativePath.toString()), true);
+            if (Files.exists(relativePath)) {
+                return Optional.of(normalizeOpenAPI(Path.of(relativePath.toString()), true));
```

Comment:
can't we simplify like this?

```suggestion
                return Optional.of(normalizeOpenAPI(relativePath, true));
```